include/sea_dsa/ShadowMem.hh

```diff
@@ -30,7 +49,24 @@ public:
 	    llvm::Pass &pass /* for dominatorTree and assumptionCache*/,
 	    bool splitDsaNodes = false, bool computeReadMod = false,
 	    bool memOptimizer = true, bool useTBAA = true);
```

Comment:
TODO: `memOptimizer` should work in the presence of function calls. I think it does. There is just an assertion because that mode was untested.